### DISCUSSION TITLE
ASUS TUF 2023 F15 (FX5072V4)  DRIVER UPDATE SECTION BUG

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1774] [General]
Describe the bug
The Driver update section is still showing RED on Realtek Lan driver even after updating driver

The bug exist since ver 0.134🙃

Clear scenario to Reproduce
Open GHELPER - Check updates section > update the driver via GHELPER given option >Download and install the driver > reopen GHELPER after restart> check update section again to see the Lan driver text in Red colour 

![IMG-20231222-WA0005](https://github.com/seerge/g-helper/assets/154586512/5f281c6a-05b0-47e7-821a-f840e95c720c)
![IMG-20231222-WA0007](https://github.com/seerge/g-helper/assets/154586512/7631c83a-5ff7-4db0-ae4e-4a79516ac556)
![IMG-20231222-WA0006](https://github.com/seerge/g-helper/assets/154586512/0461fffa-6ffb-409e-90c8-855730940e3e)

App Logs
[log.txt](https://github.com/seerge/g-helper/files/13752904/log.txt)

@seerge 👆😊

**Comment by seerge** [comment]
App just compares version of your driver and a driver on asus website, yours has lower index, so it's older. It's up to asus on how to mark their drivers

```
22-12-2023 15:56:23: Realtek LAN Driver PCI\VEN_10EC&DEV_8168&SUBSYS_208F1043 1168.010.0720.2022 vs 10.68.815.2023 = 1
22-12-2023 15:56:23: Realtek LAN Driver PCI\VEN_10EC&DEV_8168&SUBSYS_208F1043&REV_15 1168.010.0720.2022 vs 10.68.815.2023 = 1
```

**Reply by RAGE-HERO7** [reply]
I thought the app will check for the latest driver for the particular model laptop in the website automatically then user download it the install manually and after updating to lates it will show green ( The Red text is only showing for Realtek LAN driver though ).

So it's not a bug and you can't do anything about it @seerge :(

**Reply by seerge** [reply]
It does exactly that, on asus website for your model driver version is 1168.010.0720.2022 which is technically bigger than local 10.68.815.2023 as 1168 > 10. 

It's just terrible version naming on asus website (inconsistent), i can't fix it. So you will have to ignore that notification

**Reply by RAGE-HERO7** [reply]
Ok now i understand what's going on😲
Thank you for making it Clear🫡

**Reply by seerge** [reply]
Yes, this is just a typo on asus website (they missed a dot) :)
